apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/MetricOptions.java

```diff
@@ -8,13 +8,20 @@
  * class into a common location to be used here and pipeline
  */
 public final class MetricOptions implements Serializable {
+  /** The serialization key. */
```

Comment:
That's a good idea; I'll look at adding some exceptions for these, since these comments are indeed just noise.

Reply:
Added exclusions for loggers and serial UIDs. The pattern is there for anything else where the comment would be low-value (assuming we name it the same thing in each instance, since its by variable name).